CHANGELOG.md

```diff
@@ -6,15 +6,22 @@ The following is a curated list of changes in the Enact agate module, newest cha
 
 ### Added
 
+- `agate/ArcPicker` and `agate/ArcSlider` prop `disabled` to be inactive
+- `agate/ArcSlider` prop `aria-valuetext` to override `aria-valuetext` for it
 - `agate/Button` prop `iconOnly`, `iconPosition` and `minWidth`
 - `agate/Checkbox` prop `disabled`, `indeterminate`, and `indeterminateIcon`
 - `agate/CheckboxItem` prop `disabled`, `indeterminate`, `indeterminateIcon`, and `slotbefore`
 - `agate/DatePicker` prop `onSpotlightDisappear` and `spotlightDisabled`
 - `agate/Marquee` component
 - `agate/Picker` prop `noAnimation` and `wrap`
 - `agate/RangePicker` prop `noAnimation` and `wrap`
+- `agate/ThemeDecorator` context `ThemeContext` to share the context including `accent` and `highlight` properties
```

Comment:
This is just "what" you did. I think we need to add "why" this change needed here.